chapters/VK_KHR_swapchain/wsi.txt

```diff
@@ -258,7 +258,7 @@ flink:vkQueuePresentKHR any images it had already acquired from
 pname:oldSwapchain.
 E.g., an application may present an image from the old swapchain before an
```

Comment:
It is "just" a note, though I would still avoid unmarkupped RFC terminology. Should be "can:".